gempy/interpolator.py

```diff
@@ -864,6 +879,12 @@ def set_densities(self, densities):
 
             self.tg.densities.set_value(np.array(weight, dtype=self.dtype))
 
+        def set_sus(self, sus):
```

Comment:
These two parameters cannot be somehow combine in a value in the voxel? Remember that ideally the physical equation we explit it in 2:
1) The part variable with distance which therefore does not need to compute the geological model for anthing and we precompute it:
2) The part variable with the lithology, which is what we need the gological model for and therefore eventually has to be passed to the theano graph